src/main.cpp

```diff
@@ -273,10 +273,10 @@ int main(int argc, char** argv) {
     };
 
     // TODO expose a way to dump the context from the command line.
-    bool dumping_context = false;
```

Comment:
I'd prefer if this looked for an explicit flag, like `--dump-symbols`. The argc count change can stay, but the additional arguments should be checked against a flag list (which should just be this single flag for now).